TicketsDemo.XML/XmlTrainRepository.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Data.Entities;
+using TicketsDemo.Data.Repositories;
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.Serialization;
+using System.IO;
+
+namespace TicketsDemo.XML
+{
+    public class XmlTrainRepository : ITrainRepository
+    {
+        XMLSettingsService xml_set = new XMLSettingsService();
+        public List<Train> GetAllTrains()
+        {
+            
```

Comment:
please be more accuarte in code, these many whitespaces here doesn't mean anything - therefore delete them